File: src/very_easy/TemperatureConverter.swift
func toCelsius(_ fahrenheit: Double) -> Double {
  return (fahrenheit - 32) * 5 / 9
}
func toKelvin(_ celsius: Double) -> Double {
  return celsius + 273.15
}
func toFahrenheit(_ celsius: Double) -> Double {
  return celsius * 1.8 + 32
}

func convertTemperature(_ celsius: Double) -> [Double] {
  return [
    toKelvin(celsius),
    toFahrenheit(celsius),
  ]
}

File: src/main/java/very_easy/TemperatureConverter.java
package very_easy;

public class TemperatureConverter {
  public double[] convertTemperature(Double celsius) {
    return new double[] {
      toKelvin(celsius), toFahrenheit(celsius),
    };
  }

  public double toCelsius(double fahrenheit) {
    return (fahrenheit - 32.0) * 5.0 / 9.0;
  }

  public double toFahrenheit(double celsius) {
    return (celsius * 1.8) + 32.0;
  }

  public double toKelvin(double celsius) {
    return celsius + 273.15;
  }

}
